### PR TITLE
Find tabs in a file

### PR DESCRIPTION
**Pull request by tvlooy**

### CODE REVIEW

**Comment by jarv** [comment]
@tvlooy  can you update your branch from master? I just pushed a commit to add an author field to the challenges.yml so you can add your username to these (if you want).

File: challenges.yaml [hunk 1]
```diff
@@ -97,6 +97,14 @@
         - 199.37.62.156 - - [09/Jan/2017:22:42:18 +0100] "GET /posts/1/display HTTP/1.0" 200 2477
         - 251.111.109.143 - - [09/Jan/2017:22:49:02 +0100] "GET /posts/foo?appID=xxxx HTTP/1.0" 200 2477
         - 200.19.168.148 - - [09/Jan/2017:22:57:11 +0100] "GET /posts/foo?appID=xxxx HTTP/1.0" 200 3471
+  - slug: find_tabs_in_a_file
+    version: 1
+    author: cmdchallenge
```
Comment (jarv) [inline challenges.yaml]: you can put your name here (if you want) :)

Comment (tvlooy) [inline challenges.yaml]: oh, no no I'm fine :-)

**Comment by jarv** [comment]
This one might be tricky because README files are generated on the fly using the `description:`. It's currently failing in https://travis-ci.org/jarv/cmdchallenge/builds/196432131  because there is not README to check.  Maybe create `var/challenges/find_tabs_in_a_file/file-with-tabs.txt` or be sure the description is multiline with tab characters.

**Comment by jarv** [comment]
Thanks!

**Pull request merged**